### DISCUSSION TITLE
Caps Lock to toggle "Enable Creation in Hold Cells" on the fly.

### DISCUSSION THREAD

**Discussion by gab3d** [opentoonz/opentoonz#6753] [Ideas]
A Facebook user have just posted a message in the Spanish Users Group where he somewhat complained about not being able to modify a drawing from a hold cell (because a new one is created, instead of what he expected), and that post reminded me that some time ago I just happened to go through the same feeling...

Of course there is the **Enable Creation in Hold Cells** option in Preferences, that can be disabled to allow just that, but it's also true that since both behaviors are equally useful, going to Preferences and back every time seems too cumbersome to be practical, from a workflow stand point.

So, what about using an unused key (such as the Caps Lock one) to allow users to toggle that option on and off on the fly, and so having the best of both worlds? 🤔 
(somehow the "lock" part of its name, seems to help users imply that when it's activated, it will be "locking something" -in this case, the hold drawing, so a new one is not created-).

**Comment by F-Burning** [comment]
I think it's a good idea mi hermano, buen punto, but I would let the key be customizable so that block caps doesn't join the list of keys dropped in combat, like that old acquaintance called alt.

**Comment by gab3d** [comment]
@Bracket-H   ...corner case, to say the least. 😉

**Comment by gab3d** [comment]
> I think it's a good idea mi hermano, buen punto, but I would let the key be customizable so that block caps doesn't join the list of keys dropped in combat, like that old acquaintance called alt.

@F-Burning 
yes, hardcoded shortcuts are not good. 👍
